pages.ko/common/javac.md

```diff
@@ -0,0 +1,20 @@
+# javac
+
+> 자바 애플리케이션 컴파일러.
+> 더 많은 정보: <https://docs.oracle.com/en/java/javase/20/docs/specs/man/javac.html>.
+
+- `.java` 파일을 컴파일:
+
+`javac {{file.java}}`
+
+- 여러 개의 `.java` 파일들을 컴파일:
+
+`javac {{file1.java}} {{file2.java}} {{file3.java}}`
+
+- 현재 디렉토리 내의 모든 `.java` 파일들을 컴파일:
+
+`javac {{*.java}}`
+
+- `.java` 파일을 컴파일한 후, 결과 `.class` 파일을 특정 디렉토리에 위치시키기:
+
+`javac -d {{path/to/directory}} {{file.java}}`
```

Comment:
```suggestion
`javac {{파일.java}}`

- 여러 개의 `.java` 파일들을 컴파일:

`javac {{파일1.java}} {{파일2.java}} {{파일3.java}}`

- 현재 디렉토리 내의 모든 `.java` 파일들을 컴파일:

`javac {{*.java}}`

- `.java` 파일을 컴파일한 후, 결과 `.class` 파일을 특정 디렉토리에 위치시키기:

`javac -d {{경로/대상/폴더}} {{파일.java}}`
```

Ref. https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md

Reply:
Thanks for the suggestion! I will follow the common argument template from now on.